playbook/app/pb_kits/playbook/pb_selectable_list/_selectable_list.tsx

```diff
@@ -30,17 +30,18 @@ const SelectableList = (props: SelectableListProps) => {
   const classes = classnames(buildCss('pb_selectable_list_kit'), globalProps(props), className)
   const dataProps = buildDataProps(data)
   const isRadio = props.variant === "radio"
-  const defaultCheckedRadioValue = children.filter(item => item.props.defaultChecked)[0]?.props?.value
+  const defaultCheckedRadioValue = children.filter((item: {props:SelectableListItemProps} ) => item.props.defaultChecked)[0]?.props?.value
```

Comment:
```suggestion
const defaultCheckedRadioValue = children.find(
  item => React.isValidElement(item) && item.props.defaultChecked
)?.props?.value
```

Reply:
Can you see if that fixes your problem? 
I think by using `isValidElement` we can keep the children prop as React.ReactNode[]

Reply:
Tested it out locally and it did not fix it :(